src/routes/movies.ts

```diff
@@ -0,0 +1,210 @@
+import { Router, type Response, type Request } from 'express';
+import { Movie } from '../models/movie.model';
+import { moviesDataValidation } from '../validations/movies.validation';
+import { validationResult } from 'express-validator';
+
+const router = Router();
+
+/**
+ * @swagger
+ * /movies:
+ *   get:
```

Comment:
Resolver error at paths./movies.get.responses.200.content.application/json.schema.items.$ref
Could not resolve reference: Could not resolve pointer: /components/schemas/Movie does not exist in document

Reply:
in SwaggerUI

Reply:
Updated, could you please re-check